package.json

```diff
@@ -82,127 +82,127 @@
         "@louislam/ping": "~0.4.4-mod.1",
         "@louislam/sqlite3": "15.1.6",
         "args-parser": "~1.3.0",
-        "axios": "~0.27.0",
-        "axios-ntlm": "1.3.0",
+        "axios": "~0.28.1",
+        "axios-ntlm": "1.3.1",
```

Comment:
This would be still to risky given what this libary does. There might be breaking changes which happened and might make this incompatible:

> This is a helper library for NTLM Authentication using the [Axios](https://github.com/axios/axios) HTTP library on Node. It attaches interceptors to an axios instance to authenticate using NTLM for any resources that offer it.